packages/angular/projects/ui-angular/src/lib/primitives/error/__tests__/__snapshots__/error.component.test.ts.snap

```diff
@@ -34,7 +33,7 @@ exports[`amplify-error renders as expected 1`] = `
       <button
         amplify-button=""
         aria-label="Dismiss alert"
-        class="amplify-button amplify-button--link amplify-button--medium"
+        class="amplify-field-group__control amplify-alert__dismiss amplify-button amplify-button--link amplify-button--medium"
```

Comment:
Why is this necessary?

Reply:
The earlier version of Angular package did not properly resolve those extra classnames for buttons in dismissible alerts.
This change fixes that snapshot.

Reply:
What does it mean that the earlier version didn't resolve the classnames? Were they always added by jest didn't render them to a snapshot correctly, is that what you mean? Is this related to upgrading jest and testing-library?

Reply:
> This change fixes that snapshot.

Seems like it would change the rendered UI as well?

Reply:
So I looked into a little bit into this. So previously, we had an ambiguous code:

https://github.com/aws-amplify/amplify-ui/blob/41a7e90f4572d1d7e28d2d17d3be6bf42125bbac/packages/angular/projects/ui-angular/src/lib/primitives/error/error.component.html#L23-L24

`amplify-button` directive adds its own classnames (`amplify-button`, `amplify-button--link`, and `amplify-button--medium`).

And then we tried to append two additional classnames (`amplify-field-group__control`, `amplify-alert__dismiss`). 

Now the two versions (12, and 14) treat this code differently. In 12, they do not append the two new classnames. In 14, they merge the classnames.

Reply:
Thanks for looking in this @wlee221

Reply:
Makes sense. Will remove classnames `amplify-field-group__control` and `amplify-alert__dismiss` to be consistent with Angular 12.